src/pybind/mgr/cephadm/module.py

```diff
@@ -1712,6 +1712,9 @@ def describe_service(self, service_type=None, service_name=None,
         for n, spec in self.spec_store.specs.items():
             if n in sm:
                 continue
+            if n.startswith('osd.'):
+                self.log.debug("Not adding drivegroup specs  to the ServiceDescription map for now")
+                continue
```

Comment:
I think we should drop this hunk and show the osd specs.  The counts don't line up but IMO that's fine--we can  improve that soon.